pubspec.yaml

```diff
@@ -4,16 +4,16 @@ version: 3.5.1
 homepage: https://github.com/llfbandit/app_links
 
 environment:
-  sdk: ">=2.17.0 <4.0.0"
-  flutter: ">=3.0.0"
+  sdk: ">=3.2.0 <4.0.0"
+  flutter: ">=3.19.0"
 
 dependencies:
   flutter:
     sdk: flutter
 
   # https://pub.dev/packages/plugin_platform_interface
   plugin_platform_interface: ^2.0.0
-
+  web: ^0.5.1
```

Comment:
Can we relax this version constraint? For now, Flutter "stable" is aligned with version 0.4.
I would prefer web: ">=0.3.0 <0.6.0"

If so, I would recommand to lower flutter contrainst to be aligned with 0.3.0 (3.16.0).

Reply:
Yes, this should also work, as nothing in web: 0.5.0 changed for window.location.href access.

I will change it.